go-selinux/label/label_stub_test.go

```diff
@@ -75,15 +79,22 @@ func TestProcessLabel(t *testing.T) {
 }
 
 func TestCheckLabelCompile(t *testing.T) {
+	tmpFile := filepath.Join(t.TempDir(), "test_file")
+	out, err := os.OpenFile(tmpFile, os.O_WRONLY|os.O_CREATE, 0)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Why not merely use `t.TempDir()`. It returns a directory name which you can use directly as an argument to `FileLabel` and `SetFileLabel`.